packages/console/src/components/RadioGroup/Radio.module.scss

```diff
@@ -126,6 +126,36 @@
   }
 }
 
+.small {
+  position: relative;
+  border: 1px solid var(--color-border);
+  flex: 1;
+  font: var(--font-body-2);
+  height: _.unit(9);
```

Comment:
`unit` is for spacing, use `px` is fine here

Reply:
Also, we usually prefer using padding instead of height, except that you can make sure the content never wraps. Otherwise, in cases that the content wraps and if you have fixed height then the wrapped content in the next line might not be displayed properly

Reply:
Used padding instead of direct define it's size.